packages/styled-system/tests/chromatic/ThemeProvider.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import { HopperProvider, useColorSchemeContext } from "../../src/index.ts";
+import { useEffect } from "react";
+import type { Meta, StoryObj } from "@storybook/react";
+import { Div } from "@hopper-ui/styled-system";
+
+export default {
+    title: "Chromatic/ThemeProvider",
```

Comment:
We used to have the Chromatic for chromatic stories. I guess it doesn't make much sense anymore since chromatic is going to be used mainly as a visual regression tool